app/helpers/language_helper.py

```diff
@@ -13,17 +14,18 @@
 }
 
 
-def handle_language() -> None:
+def handle_language(metadata: Optional[Mapping[str, Any]] = None) -> None:
+    metadata = metadata or get_metadata(current_user)
     session_store = get_session_store()
+    schema_name = metadata.get("schema_name") if metadata else None
```

Comment:
You could set metadata to empty dict and avoid the `if`.
```py
metadata = metadata or get_metadata(current_user) or {}
schema_name = metadata["schema_name"]
```
Also used bracket access as `schema_name` is currently mandatory. If we had type hints, this would error at the moment. When we come to have schema_name/schema_url, this should break during dev which will help. Non blocker as this will change in the next card anyway as we will have a metadata proxy.